### PR TITLE
Use the random_name function on the space_name argument of the space …

### LINKED ISSUE #6380: Use the `random_name` function on the `space_name` argument of the `space create` command
## Current behavior

The `space_name` argument of the `ockam space create` command is calling `hex::encode()` to generate the random name in the `default_value_t` attribute.

## Desired behavior

The `space_name` argument of the `ockam space create` command should be calling the [`random_name()`](https://github.com/build-trust/ockam/blob/75fa2e3ece6c8ece4a89056e5360d3b1ca19ddeb/implementations/rust/ockam/ockam_api/src/cli_state/mod.rs#L421) function.

## Implementation details

Don't forget to import the `random_name()` function with `use ockam_api::cli_state::random_name;`.

This is the argument that must be changed, where `default_value_t = random_name()`:

https://github.com/build-trust/ockam/blob/140a8bd71183754d70ec1e4bc1a5c962062fa0a9/implementations/rust/ockam/ockam_command/src/space/create.rs#L25-L26

---

<i>

Some tips for newcomers:
1. Create a new branch for the issue you are working on (don't push directly to your fork's `develop` branch)
2. Run `cargo fmt`. You can configure the editor to run it automatically everytime a file is saved
3. Run `cargo clippy` to make sure the code compiles + follows rust standards/best practices
4. You don't need to keep your branch updated with `develop` until there is a merge conflict. If you still want to update it, use `git rebase develop` instead of `gir merge develop` 

We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).

</i>

### PR DESCRIPTION
**Pull request by dhruvmehtaaa**
Fixes #6380 

## Current behavior

The space_name argument of the ockam space create command is calling hex::encode() to generate the random name in the default_value_t attribute.

## Proposed changes
The space_name argument of the ockam space create command should be calling the random_name() function.

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] There are no Merge commits in this Pull Request. Ockam repo maintains a linear commit history. We merge Pull Requests by rebasing them onto the develop branch. Rebasing to the latest develop branch and force pushing to your Pull Request branch is okay.
- [x] I have read and accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have read and accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/.github/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a separate Pull Request, this will mark the commit as verified.

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/space/create.rs [hunk 1]
```diff
@@ -22,7 +22,7 @@ const AFTER_LONG_HELP: &str = include_str!("./static/create/after_long_help.txt"
 )]
 pub struct CreateCommand {
     /// Name of the space - must be unique across all Ockam Orchestrator users.
-    #[arg(display_order = 1001, value_name = "SPACE_NAME", default_value_t = hex::encode(&random::<[u8;4]>()), hide_default_value = true, value_parser = validate_space_name)]
+    #[arg(display_order = 1001, value_name = "SPACE_NAME", default_value_t = hex::encode(&random::<[u8;4]>()), hide_default_value = true, value_parser = validate_space_name)]
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/src/space/create.rs]: I think you forgot to actually use the `random_name` function right? You need to update the `default_value_t` attribute.

Comment (dhruvmehtaaa) [reply implementations/rust/ockam/ockam_command/src/space/create.rs]: I have committed the change, can you please review it now

**Review by adrianbenavides — COMMENTED** [review]
Changes look good now but you need to run `cargo fmt` and `cargo clippy` to fix a couple of issues. After that, it will be ready to merge 👍

**Review by adrianbenavides — APPROVED** [review]
Thanks @dhruvmehtaaa , will get merged soon! 🚀

**Review by etorreborre — APPROVED** [review]

**Pull request merged**